COMPILING.md

```diff
@@ -0,0 +1,143 @@
+## Points to note
+
+- Jamulus can be compiled for Linux, Windows and macOS. However, the preferred method of supporting these platforms is to use the binaries generated by the autobuild process in the Jamulus repository.
+- For unattended installs, see the contributed [installation scripts](https://github.com/jamulussoftware/installscripts)
+- There are reports from people who successfully compile and run Jamulus on BSDs.
+- Android and iOS are not officially supported, therefore there are no instructions yet.
+
+---
+
+
+## Download sources
+
+For .tar.gz [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.tar.gz) to download the latest release
+
+For .zip [use this link](https://github.com/jamulussoftware/jamulus/archive/master.zip)
+
+## Linux
+
+#### Install dependencies
+
+On Ubuntu-based distributions 18.04+, Debian 9+ or 10 and Raspberry Pi Raspbian Buster release or later:
+
+* build-essential
+* qt5-qmake
+* qtdeclarative5-dev
+* qt5-default
+* qttools5-dev-tools
+* libjack-jackd2-dev
+
+#### On Fedora
+
+* qt5-qtdeclarative-devel
+* jack-audio-connection-kit-dbus
+* libqt5concurrent5 (qt5-linguist for Fedora 33)
+* jack-audio-connection-kit-devel
+
+#### For all desktop distributions
+
+[QjackCtl](https://qjackctl.sourceforge.io/) is optional, but recommended to configure JACK.
+
+
+### Standard desktop build
+
+```
+make distclean
+qmake ;# qmake-qt5 on Fedora 33
+make
+sudo make install
+```
+
+`make distclean` is optional but ensures a clean build environment. `make install` is optional and puts the Jamulus binary into `/usr/local/bin`.
+
+**Users of Raspberry Pi** You may want to compile the client on another machine and run the binary on the Raspberry Pi. In which case the only libraries you need to run it are those for a [headless server](Server-Linux#running-a-headless-server) build, but _with_ the JACK sound packages. 
```

Comment:
I still don't understand why we need this and would suggest removing this paragraph.
We don't provide documentation for cross-compilation (yet?), so this suggestion makes little sense to me as I'd argue that most of the people are still mainly using x86_64 machines (maybe it'll change with Mac M1 which is at least ARM, but I'm not sure whether this is the same flavor (ARM version, 32/64bit, armhf).

Reply:
Yes, I think this was out of a desire to explain things to people inexperienced with compilation perhaps, not sure. I'll remove it as too noisy.